wporg-gp-translation-events.php

```diff
@@ -250,7 +250,8 @@ public function submit_event_ajax() {
 			if ( ! $event || self::CPT !== $event->post_type ) {
 				wp_send_json_error( esc_html__( 'Event does not exist.', 'gp-translation-events' ), 404 );
 			}
-			if ( ! ( $can_crud_event || current_user_can( 'edit_post', $event_id ) || intval( $event->post_author ) === get_current_user_id() ) ) {
+			$can_edit = $can_crud_event && current_user_can( 'edit_post', $event_id ) && intval( $event->post_author ) === get_current_user_id();
```

Comment:
I think with this suggestion the post author can no longer edit the post when they are the post author.

Reply:
Great catch, will do the author check first.